.github/workflows/stale.yaml

```diff
@@ -0,0 +1,36 @@
+# This workflow warns and then closes issues and PRs that have had no 
+# activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see: https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '0 4 * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: >
+          This issue hasn't been updated for a while, marking as stale, 
+          please respond within the next 7 days to remove this label.
+        stale-pr-message: >
+          This PR hasn't been updated for a while, marking as stale.
+        stale-issue-label: stale
+        stale-pr-label: stale
+        days-before-stale: 90
+        days-before-close: 7
+        exempt-issue-labels: accepted
+        exempt-pr-labels: accepted
+        remove-issue-stale-when-updated: true
+        remove-pr-stale-when-updated: true
```

Comment:
Please add enter on line 36. Git diff is not great a handling files without line breaks at the end.

Reply:
Fixed.